### DISCUSSION TITLE
how to disable bufferline tabs attachment to buffers

### DISCUSSION THREAD

**Discussion by cofeek-codes** [LazyVim/LazyVim#4270] [Q&A] [answered]
when i split window and attach other file to buffer, bufferline creates new tab for it. i don't want it. i am creating tabs with custom keymap that calls `:tabedit<Return>`
![2024-08-07-175901_1920x1080_scrot](https://github.com/user-attachments/assets/115b832e-5440-4dd6-970a-859e780cf992)

**Comment by dpetka2001** [comment]
What you're seeing is the way `bufferline` shows the buffers, so that it's more consistent with the way other editors look. These are not Neovim tabs. Neovim tabs show on the far right corner of the tabline. Read the plugin's docs to see if it provides a way to disable this behavior or disable the plugin as a whole if you don't like it.

**Comment by cofeek-codes** [comment, marked as answer]
UPDATE: i achieved it using `mode = "tabs"`
```lua
return {
  {
    "akinsho/bufferline.nvim",
    event = "VeryLazy",
    keys = {
      { "<Tab>", "<Cmd>BufferLineCycleNext<CR>", desc = "Next tab" },
      { "<S-Tab>", "<Cmd>BufferLineCyclePrev<CR>", desc = "Prev tab" },
    },
    opts = {
      options = {
        mode = "tabs",
        show_buffer_close_icons = false,
        show_close_icon = false,
      },
    },
  },
}

```
